runtime/onert/core/include/ir/train/TrainableGraph.h

```diff
@@ -101,6 +101,7 @@ class TrainableGraph : public IGraph
   void changeDerivativeShape(const OperandIndex &ind, const ir::Shape &new_shape);
   void addInput(const OperandIndex &ind, const std::string &name = "");
   void addOutput(const OperandIndex &ind, const std::string &name = "");
+  void addLoss(const IOIndex &io_ind, const OperandIndex &ind);
```

Comment:
How about changing the name of the parameter to make it easier to know the index for which operand in Loss?

Reply:
```suggestion
  void addLoss(const IOIndex &out_ind, const OperandIndex &loss_ind);
```

How about this?

Reply:
I mean the name of operand. How about this?
```suggestion
  void addLoss(const IOIndex &true_io_ind, const OperandIndex &true_ind);
```

Reply:
I fixed this code to `pred_io_ind` and `loss_ind`.

The loss map has a combination of `pred_io_ind` and the loss output index of loss op for `pred_io_ind` output.